core/src/main/java/org/mskcc/cbio/portal/dao/DaoGenericAssay.java

```diff
@@ -85,4 +85,29 @@ public static void deleteGenericEntityPropertiesByStableId(String stableId) thro
             JdbcUtil.closeAll(DaoGeneticEntity.class, con, pstmt, rs);
         }
     }
+
+    public static boolean shouldGenericAssayMetaBeDeleted(int cancerStudyId) throws DaoException {
```

Comment:
We are testing that genetic entities only exist in this cancer study.  Maybe we should rename this routine - geneticEntitiesOnlyExistInSingleStudy